src/genn/backends/cuda/backend.cc

```diff
@@ -507,6 +507,9 @@ std::string Backend::getNVCCFlags() const
     if(getPreferences<Preferences>().generateLineInfo) {
         nvccFlags += " --generate-line-info";
     }
+    if(getPreferences<Preferences>().enableNVTX) {
+        nvccFlags += " -lnvToolsExt";
+    }
```

Comment:
According to the documentation, NVTX has been header-only since CUDA 10 so this is unnecessary (which is good because I can't figure out the windows equivalent!). Would be good to add an error check for CUDA < 10 and this being enabled in the CUDA backend constructor though.